dspace/config/submission-forms.xml

```diff
@@ -152,6 +155,9 @@
                         it, please enter the types and the actual numbers or codes.
                     </hint>
                     <required></required>
+<!--                    <acl>-->
+<!--                        policy=allow,action=read,grantee-type=user,grantee-id=*-->
```

Comment:
why?

Reply:
Removed

Reply:
Added example acl definition to the form definition file